cerberus/util.py

```diff
@@ -11,9 +27,13 @@ def throw_if_bad_response(response):
     """Throw an exception if the Cerberus response is not successful."""
     try:
         response.raise_for_status()
-    except RequestException as e:
-        msg = 'Response code: {}; response body:\n{}'.format(response.status_code, json.dumps(response.json(), indent=2))
-        raise CerberusClientException(msg)
+    except RequestException:
+        try:
+            msg = 'Response code: {}; response body:\n{}'.format(response.status_code, json.dumps(response.json(), indent=2))
+            raise CerberusClientException(msg)
+        except ValueError:
+            msg = 'Response code: {}; response body:\n{}'.format(response.status_code, response.text)
```

Comment:
Will `response.text` work?

I thought the entire body returned was HTML when the user is rate limited?

Reply:
Yes it works on HTML